config/urls.yaml

```
@@ -0,0 +1,34 @@
+
+# This file contains an array of objects,
+# each having sets of URLs to the equivalent article/video in separate hosting locations.
+# This information is useful when mapping or converting one URL to another,
+# such as when generating a version of an article to be hosted on a specific platform
+
```

YAMLLINT suggests --- as "document start" here.
```suggestion
---
```